www/xmlResult.php

```diff
@@ -88,6 +88,8 @@
                 echo "<latency>{$test['testinfo']['latency']}</latency>\n";
                 echo "<plr>{$test['testinfo']['plr']}</plr>\n";
             }
+            if( @strlen($test['testinfo']['mobile']) )
```

Comment:
The mobile flag is currently an integer, not a string: https://github.com/WPO-Foundation/webpagetest/blob/master/www/runtest.php#L170

Maybe just check isset and pass the value through?  That way it will cover it if I extend it to allow for multiple mobile profiles.